hw02/EnemyCollision.cs

```diff
@@ -7,34 +7,50 @@ public class EnemyCollision : MonoBehaviour
     // Start is called before the first frame update
     void Start()
     {
-        
+
     }
 
     // Update is called once per frame
     void Update()
     {
-        
+
     }
-    
+
     // TODO: Problem 7: Why did I not include the words public or private here?
     // What does it mean when I only write void (Movement)? What does void mean?
-    void Movement ()
+
+    //The reason you did not include the words public or private here is that the method defaults to private if you did not specify it.
+    //If you only write void(Movement), you will get no return value. Void means the method does not return any value. 
+    void Movement()
     {
         // TODO: Problem 8: Please explain what the next 4 lines mean.
-        // What is Input.GetAxis, transform.Translate, and transform.Rotate? 
+
         float forwardMovement = Input.GetAxis("Vertical") * speed * Time.deltaTime;
+        //This function reads player input from the vertical axis, and ensures the player’s forward/backward movement is smooth and frame-rate independent.
         float turnMovement = Input.GetAxis("Horizontal") * turnSpeed * Time.deltaTime;
-        
+        //Similar to the vertical axis, this reads the horizontal input, and makes the turning speed consistent across frames.
+
         transform.Translate(Vector3.forward * forwardMovement);
+        //This moves the object forward by a certain amount.
         transform.Rotate(Vector3.up * turnMovement);
+        //This rotates the object around the Y-axis (which is vertical in 3D space), based on the horizontal input.
+
+        // What is Input.GetAxis, transform.Translate, and transform.Rotate? 
+        //Input.GetAxis reads player input for movement.
```

Comment:
Close! Input.GetAxis will return -1 or 1 depending on the input. Translate will move the object with x, y, z. Rotate will rotate objects with x,y,z axis.